cliquet/__init__.py

```diff
@@ -2,8 +2,8 @@
 """
 import warnings
 import pkg_resources
-
```

Comment:
Here we separate imports from the sdlib from imports from other libraries having ``import`` first and then ``from``